### PR TITLE
Decorate Event As passing/failing the Cleaning Cut

### PR DESCRIPTION
**Pull request by johnda102**

### CODE REVIEW

File: xAODAnaHelpers/JetSelector.h [hunk 1]
```diff
@@ -59,6 +59,8 @@ class JetSelector : public xAH::Algorithm
    */
 
   bool m_cleanEvent;
+  /** @brief Make event with decorator if any passing jets are not clean */
```
Comment (kratsg) [inline xAODAnaHelpers/JetSelector.h]: Shouldn't this say "Mark event with decoration"?

**Comment by johnda102** [comment]
go ahead

**Comment by kratsg** [comment]
@johnda102 - the bot is currently down. Waiting on Lincoln to bring it up when he wakes up.

**Comment by kratsg** [comment]
go ahead

**Pull request merged**